src/pages/SettingsPage.js

```diff
@@ -86,6 +113,25 @@ const SettingsPage = ({
                         {Str.removeSMSDomain(session.email)}
                     </Text>
                     )}
+                    <View style={[styles.settingsPageBody, styles.mt4]}>
+                        <Text style={[styles.textLabel, styles.textStrong, styles.mt2, styles.mb2]} numberOfLines={1}>
```

Comment:
Okay apologies for the late feedback here, but I actually think we also want the label color to use `themeColors.heading`

So that being said, seems like we have a lot of styles we're chaining together here and I think we will be reusing this in many other places. So I think it makes sense to make a new style like `styles.formLabel` that has:

```
color: themeColors.heading,
fontFamily: fontFamily.GTA_BOLD,
fontSize: variables.fontSizeLabel,
fontWeight: fontWeightBold,
```

Thoughts?

Reply:
I think this is a good idea. I can see this being re-used once the menus are expanded with more options in issue 1493
and maybe for the user details labels in the new profile view on 1492.

Reply:
We already have a styles.formLabel implemented:
```
    formLabel: {
        color: themeColors.text,
        fontSize: variables.fontSizeLabel,
        lineHeight: 18,
        marginBottom: 8,
    },
```
It looks like this in this situation:
![image](https://user-images.githubusercontent.com/47184118/109486666-15c91d80-7a7b-11eb-891b-f58ab5e94881.png)

So I've made a new one named settingsLabel. Whoever said naming things is hard was right :)
In the other label style the margin was in the style. I think it's likely you're going to always want a margin below the label. We could add `marginBottom: 8`
What do you think?
Here is the result of the changes:
![image](https://user-images.githubusercontent.com/47184118/109487145-bd465000-7a7b-11eb-9cde-258141c6f4f2.png)

Reply:
Ah, I say we just reuse the existing `formLabel` style and just add `fontFamily: fontFamily.GTA_BOLD, fontWeight: fontWeightBold,` to it. This way we only have one global formLabel style.

Reply:
do you want the text colour for the label style to be text or heading? It's currently text.

Additionally form label is implemented in other places such as changing the login page to:
![loginscreen change](https://user-images.githubusercontent.com/47184118/109536433-a0c70980-7ab5-11eb-890d-a25edded934c.png)
compared to before:
![loginscreen change](https://user-images.githubusercontent.com/47184118/109536570-cc49f400-7ab5-11eb-8e9c-1722dea131ca.png)
All the usages are as follows:
![image](https://user-images.githubusercontent.com/47184118/109536674-ec79b300-7ab5-11eb-9882-360a0c91a7a4.png)

Reply:
Yup, let's use `themeColors.heading` and it is okay if this updates everywhere. Thanks!

Reply:
ok I've pushed these changes to the branch 👍

Reply:
Awesome, thanks Anthony!